From 6e4ebe9bc913e3eee2935050c12bc1ed9e968cf4 Mon Sep 17 00:00:00 2001
From: Michele Mammini <mamminim@gmail.com>
Date: Thu, 4 Jul 2013 17:05:01 +0200
Subject: [PATCH] Edit Readme

---
 README.md | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/README.md b/README.md
index 6147bc4..802c820 100644
--- a/README.md
+++ b/README.md
@@ -1,6 +1,9 @@
+>> Pronto!
 dbmng
 =====
 
 database manager php
 test!
 Test2
+
+IO sono Michele
\ No newline at end of file
-- 
1.8.3.2

